target/linux/kirkwood/image/Makefile

```diff
@@ -11,7 +11,7 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
 KERNEL_LOADADDR:=0x8000
-TARGET_DEVICES = linksys-audi linksys-viper dockstar goflexnet goflexhome iconnect pogo_e02 ib62x0 nsa310b nsa325 on100
+TARGET_DEVICES = linksys-audi linksys-viper dockstar goflexnet goflexhome iconnect pogo_e02 pogo_v4 ib62x0 nsa310b nsa325 on100
```

Comment:
Your branch misses the latest code from master. This line was removed/reworked on 12. Dec 2017. Please rebase your fork to the latest code from master.

Reply:
These commands should update your branch:

```
git remote add upstream "https://github.com/lede-project/source.git"
git checkout pogoplug-v4
git pull --rebase upstream master
```